evalml/pipelines/component_graph.py

```diff
@@ -445,7 +450,10 @@ def _transform_features(
         if not fit:
             X_schema = (
                 self._return_non_engineered_features(X).ww.schema
-                if "DFS Transformer" in self.compute_order
+                if any(
+                    isinstance(c, DFSTransformer)
+                    for c in self.component_instances.values()
+                )
```

Comment:
My only thought here - we keep running into this and having to change the code of how we check in all sorts of different places. I don't think it needs to be changed for this PR, but I think we should consider switching this over to either a method or attribute of the component graph itself, so we can easily just check `ComponentGraph.has_dfs` from anywhere.

Reply:
yeah good call - I'll just add it here.